src/makeInfinityAdder.js

```diff
@@ -4,7 +4,22 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let totalSum = 0;
+
+  const adder = (a) => {
```

Comment:
Consider adding a check to ensure that the input `a` is a number. If `a` is not a number, it could lead to unexpected behavior when adding to `totalSum`.